way2.js

```diff
@@ -0,0 +1,42 @@
+    title: "Title Three",
+    subheading: 'posted on 29 June //Category',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quibusdam vitae sitreiciendis excepturimaiores quam rem dolorum ea nulla aut laborum placeat facere ullam odit qui est non deseruntdoloribus?'
+
+},
+
+]
+
+for (let i = 0; i <= data.length; i++) {
+    let div = document.createElement("div");
+    div.innerHTML = `<div class="elements-container">
+<div class="image-container"><img
+        src= ${data[i].url} /></div>
+<div class="content-container">
+    <h2 class="content-title">${data[i].title}</h2>
+    <p class="subheading">${data[i].subheading}</p>
+    <p class="description">${data[i].description}</p>
+    <button id="continue-btn">Continue Reading</button>
+</div>
+</div>
+<hr>`;
+    main.appendChild(div);
+}
+
+
+
```

Comment:
only one extra lines should be added at the end of the file.
Remove the extra ones.